analyzers/src/SonarAnalyzer.CSharp/Rules/Hotspots/ClearTextProtocolsAreSensitive.cs

```diff
@@ -117,22 +119,22 @@ public ClearTextProtocolsAreSensitive(IAnalyzerConfiguration analyzerConfigurati
                 }
 
                 context.RegisterSyntaxNodeActionInNonGenerated(VisitStringExpressions, SyntaxKind.StringLiteralExpression, SyntaxKind.InterpolatedStringExpression);
-                context.RegisterSyntaxNodeActionInNonGenerated(VisitObjectCreation, SyntaxKind.ObjectCreationExpression);
+                context.RegisterSyntaxNodeActionInNonGenerated(VisitObjectCreation, SyntaxKind.ObjectCreationExpression, SyntaxKindEx.ImplicitObjectCreationExpression);
```

Comment:
Shouldn't the title of this PR mention S5332 as well? This new registration is a new feature implementation.